src/app/components/event-page/event-page.component.html

```diff
@@ -23,7 +23,7 @@
                 </div>
             </div>
             <div class="fee-icon-row">
-                <div class="event-fee">{{event.fee}} lei</div>
+                <div class="event-fee">{{event.fee === 0 ? "Free" : event.fee + " lei"}}</div>
```

Comment:
For future reference, you can use event.fee ? "Free" : event.fee + " lei" and get rid of that _=== 0_.
Also as a remark if we're already here, "===" checks not only value but type too.
So 
0 == "0" --> returns true (because it doesn't check the type)
0 === "0" --> returns false (because  one is int and the other is string)
Just to be aware in the future in case you didn't know :)